src/coreclr/src/pal/tests/palsuite/README.txt

```diff
@@ -44,14 +44,14 @@ The results are logged to %ROTOR_DIR%\tests\pvtResults.log.
 
 3. ENVIRONMENT VARIABLES AND AUTOMATED TESTING SPECIFICS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    
+
 3.1 PAL_DISABLE_MESSAGEBOX
 
-For automated testing on the WIN32 PAL, the following environment
+For automated testing on the CLR_CMAKE_TARGET_WIN32 PAL, the following environment
```

Comment:
I believe this rename was unintentional. However, this whole paragraph is obsolete, there is no PAL_DISABLE_MESSAGEBOX anymore. Since you are at it, could you please delete the paragraph?

Reply:
Thanks, fixed. My glob pattern was bit permissive (operated on `*.txt` and `*.cmake` instead of `CMakeLists.txt`) and string matched the replace pattern. Maybe this file can be changed to markdown (`.md`) separately.